### PR TITLE
Added Decimal to Octal Conversion

### LINKED ISSUE #5508: Decimal to Octal in c language using recursion
## 💥 Proposal

This program generally converts decimal number into octal number
Input: 162
Output: 242
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

Yes I have
Please assign me

### PR DESCRIPTION
**Pull request by UG-SEP**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

yes, I have 

### Description

I added decimal to octal conversion program in c->math .This program convert decimal number into octal

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
closes: #5508 
Please review it @HarshCasper and @siddharth25pandey

### CODE REVIEW

File: C/math/Decimal to Octal.c [hunk 1]
```diff
@@ -0,0 +1,51 @@
+// Program to convert decimal to octal
+#include<stdio.h>
```
Comment (UG-SEP) [inline C/math/Decimal to Octal.c]: should i add math.h header file or not

File: C/math/Decimal to Octal.c [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+/*
+Example:
+case 1:
+Input: Enter a Number: 16
+Output: The octal conversion of 16 is: 20
+case 2:
+Input: Enter a Number: -10
+Output: The octal conversion of -10 is -12
+
+Time complexity:O(n)
```
Comment (UG-SEP) [inline C/math/Decimal to Octal.c]: is this time complexity is correct or not

**Comment by UG-SEP** [comment]
@HarshCasper @ankitaggarwal23 @siddharth25pandey please review it

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Few changes: 

- Add Multi-Line Comments 
- Add a beginner-friendly description at the top
- Use proper code Format

**Comment by UG-SEP** [comment]
> * Add Multi-Line Comments

I have added Multi-Line comments where needed

**Comment by UG-SEP** [comment]
> * Use proper code Format
  
 @ankitaggarwal23  can you tell me what is improper in code so i can change  that

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Use proper code Format

**Comment by UG-SEP** [comment]
> Use proper code Format

 let me know @ankitaggarwal23

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
- Use proper code Format

**Review by ankitaggarwal23 — COMMENTED** [review]
Use proper code Format

**Pull request closed without merging**